### DISCUSSION TITLE
Updating @discordjs/voice dependency.

### DISCUSSION THREAD

**Discussion by zoltan-lima** [Androz2091/discord-player#1677] [Q&A] [answered]
Hello,
I would like to continue using discord-player 5.4.0 instead of migrating to v6. However, I can not update my @discordjs/voice dependency as it is set to v0.11.0 in the package.json. I need to update due to some breaking changes on my bot and @discordjs/voice is the only package I would like to update. I have @discordjs/voice@0.15.0 installed however discord-player continues to pick the v0.11.0 specified in package.json.
How would I go about explicitly updating the package in discord-player to the latest version?

**Comment by twlite** [comment, marked as answer]
Use the overrides field in package.json as mentioned in https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides
